pkg/netlink/fake/netlink.go

```diff
@@ -235,16 +232,6 @@ func (n *NetLink) EnableLooseModeReversePathFilter(interfaceName string) error {
 }
 
 func (n *NetLink) ConfigureTCPMTUProbe(mtuProbe, baseMss string) error {
```

Comment:
Shouldn't we remove this function altogether ? or is it called somewhere?

Reply:
it is used in global  gateway monitor.

Reply:
Then we should not delete code, right.

Reply:
No, I just reverted the changes the MTU probe PR did. This was a function returning nil before those changes. I am not sure deleting it will have any impact in globalnet UT.